GPipe-GLFW/src/Graphics/GPipe/Context/GLFW/Handler.hs

```diff
@@ -117,6 +118,15 @@ defaultHandleConfig = HandleConfig
         }
```

Comment:
I think `defaultHandleConfig` needs to have a field set with major minor versions 3 and 3. Current users of `defaultHandleConfig` would crash if your change was merged because the field is not initialized.

Reply:
Shame on me! I've must have been blind to no see my editor's not so subtle warning regarding this omission.